vm/systemSmartContracts/validator.go

```diff
@@ -1730,7 +1730,6 @@ func (v *validatorSC) getTotalStakedTopUpStakedBlsKeys(args *vmcommon.ContractCa
 	topUp := big.NewInt(0).Set(registrationData.TotalStakeValue)
 	topUp.Sub(topUp, stakeForNodes)
 	if topUp.Cmp(zero) < 0 {
-		log.Warn("topup value is less than 0")
```

Comment:
Maybe log.Debug instead deletion

Reply:
no need for this log. as we can see the results of this in several other places.